xchange-coinmate/src/main/java/info/bitrich/xchangestream/coinmate/dto/auth/PusherAuthParamsObject.java

```diff
@@ -15,21 +18,33 @@
 
 public class PusherAuthParamsObject {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(PusherAuthParamsObject.class);
```

Comment:
`LOGGER` is the correct convention for a `static final`.

Reply:
yes, i put the log in order to be consistent with @Log tag on lombok. I can change it if you like